zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/ReconfigBackupTest.java

```diff
@@ -93,7 +83,7 @@ public void testBackupStatic() throws Exception {
             mt[i] = new MainThread(i, clientPorts[i], currentQuorumCfgSection, false);
             // check that a dynamic configuration file doesn't exist
             assertNull("static file backup shouldn't exist before bootup", mt[i].getFileByName("zoo.cfg.bak"));
-            staticFileContent[i] = getFileContent(mt[i].confFile);
+            staticFileContent[i] = FileUtils.readFileToString(mt[i].confFile, StandardCharsets.UTF_8);
```

Comment:
You seem to have a lot of little open PRs for the same action in different parts of the code. It is making reviewing a bit redundant. See my comments on your other PR with my suggestions to use static imports for `UTF_8` for readability, and the Java-native options for reading files without needing to use Commons.